queries.yaml

```diff
@@ -280,13 +280,13 @@ layers:
       - vectordatasource.transform.major_airport_detector
       - vectordatasource.transform.elevation_to_meters
       - vectordatasource.transform.normalize_operator_values
-      - vectordatasource.transform.truncate_min_zoom_to_2dp
+      - vectordatasource.transform.truncate_min_zoom_to_1dp
     sort: vectordatasource.sort.pois
     area-inclusion-threshold: 1
   boundaries:
     geometry_types: [Polygon, MultiPolygon, LineString, MultiLineString]
     simplify_before_intersect: true
-    simplify_start: 8
+    simplify_start: 0
```

Comment:
@peitili this change is since PR review, based on build QA. The feature merging was working, but the linework detail was significantly more detailed that the water or earth layers on visual inspection. See PR comment for images.

Reply:
Since boundaries appear in the 0/0/0 tile and zoom 1 tiles, this will have largest impact there on p50 size (not just p99).